blockbuster/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "blockbuster"
 description = "Metaplex canonical program parsers, for indexing, analytics etc...."
-version = "2.3.0"
+version = "2.4.0"
```

Comment:
I raised the crate version, but I am not sure that it's necessary. So if it's redundant plz inform me